### PR TITLE
Fixing a few more build warnings

### PR DESCRIPTION
**Pull request by bergerhoffer**
Fixing these build warnings:

```
asciidoctor: WARNING: modules/identity-provider-htpasswd-update-users.adoc: line 66: unterminated listing block
asciidoctor: WARNING: modules/osdk-bundle-operator.adoc: line 16: section title out of sequence: expected level 3, got level 4
asciidoctor: WARNING: modules/osdk-deploy-olm.adoc: line 16: section title out of sequence: expected level 3, got level 4
asciidoctor: WARNING: modules/osdk-bundle-operator.adoc: line 16: section title out of sequence: expected level 3, got level 4
asciidoctor: WARNING: modules/osdk-deploy-olm.adoc: line 16: section title out of sequence: expected level 3, got level 4
asciidoctor: WARNING: modules/osdk-bundle-operator.adoc: line 16: section title out of sequence: expected level 3, got level 4
asciidoctor: WARNING: modules/osdk-deploy-olm.adoc: line 16: section title out of sequence: expected level 3, got level 4
asciidoctor: WARNING: modules/migration-mtc-cr-manifests.adoc: line 35: no callout found for <1>
asciidoctor: WARNING: modules/migration-updating-deprecated-internal-images.adoc: line 64: no callout found for <1>
```

Preview:
* https://deploy-preview-33876--osdocs.netlify.app/openshift-enterprise/latest/operators/operator_sdk/golang/osdk-golang-tutorial.html#osdk-bundle-deploy-olm_osdk-golang-tutorial
* https://deploy-preview-33876--osdocs.netlify.app/openshift-enterprise/latest/operators/operator_sdk/ansible/osdk-ansible-tutorial.html#osdk-bundle-deploy-olm_osdk-ansible-tutorial
* https://deploy-preview-33876--osdocs.netlify.app/openshift-enterprise/latest/operators/operator_sdk/helm/osdk-helm-tutorial.html#osdk-bundle-deploy-olm_osdk-helm-tutorial
* https://deploy-preview-33876--osdocs.netlify.app/openshift-enterprise/latest/migrating_from_ocp_3_to_4/advanced-migration-options-3-4.html#migration-mtc-cr-manifests_advanced-migration-options-3-4
* https://deploy-preview-33876--osdocs.netlify.app/openshift-enterprise/latest/migrating_from_ocp_3_to_4/troubleshooting-3-4.html#migration-updating-deprecated-internal-images_troubleshooting-3-4
* https://deploy-preview-33876--osdocs.netlify.app/openshift-enterprise/latest/authentication/identity_providers/configuring-htpasswd-identity-provider.html#identity-provider-htpasswd-update-users_configuring-htpasswd-identity-provider

### CODE REVIEW

File: modules/migration-mtc-cr-manifests.adoc [hunk 1]
```diff
@@ -28,7 +28,7 @@ spec:
   destMigClusterRef:
     name: <destination_cluster_ref>
     namespace: openshift-migration
-  namespaces:
+  namespaces: <1>
```
Comment (bergerhoffer) [inline modules/migration-mtc-cr-manifests.adoc:31]: @apinnick Can you confirm if these callouts (here and in the below file) have been added at the correct location?

File: operators/operator_sdk/ansible/osdk-ansible-tutorial.adoc [hunk 2]
```diff
@@ -38,8 +38,8 @@ include::modules/osdk-run-deployment.adoc[leveloffset=+2]
 [id="osdk-bundle-deploy-olm_{context}"]
 === Bundling an Operator and deploying with Operator Lifecycle Manager
 
-include::modules/osdk-bundle-operator.adoc[leveloffset=+4]
-include::modules/osdk-deploy-olm.adoc[leveloffset=+4]
+include::modules/osdk-bundle-operator.adoc[leveloffset=+3]
```
Comment (bergerhoffer) [inline operators/operator_sdk/ansible/osdk-ansible-tutorial.adoc:41]: @adellape Can you confirm that these leveloffset adjustments are correct (here and below 2 files)?

File: modules/migration-mtc-cr-manifests.adoc (hunk 1 above)
Comment (apinnick) [reply modules/migration-mtc-cr-manifests.adoc:31]: @bergerhoffer 
Yes, this is correct.

File: operators/operator_sdk/ansible/osdk-ansible-tutorial.adoc (hunk 2 above)
Comment (adellape) [reply operators/operator_sdk/ansible/osdk-ansible-tutorial.adoc:41]: LGTM

**Pull request merged**